### DISCUSSION TITLE
bc-fips 2.0.0 examples not accessible from site

### DISCUSSION THREAD

**Discussion by rottemp** [bcgit/bc-java#1919] [Q&A]
When opening https://www.bouncycastle.org/documentation/documentation-java/#java-fips-test-packages there are links to examples that are not working.
Are there any examples for using the bc-fips version 2.0.0 and up?
Thank you

**Comment by dghgit** [comment]
These should now be working. Apologies for the inconvenience, we are in the process of migrating the website.
